deployments/operator/manifests/bases/intel-device-plugins-operator.clusterserviceversion.yaml

```
@@ -5,9 +5,11 @@ metadata:
     alm-examples: '[]'
     capabilities: Basic Install
     categories: Drivers and plugins
-    containerImage: intel/intel-deviceplugin-operator:0.21.0
-    createdAt: "2021-08-18"
-    description: This One Operator is a Kubernetes custom controller whose goal is to serve the installation and lifecycle management of Intel device plugins for Kubernetes.
+    containerImage: intel/intel-deviceplugin-operator:0.23.0
+    createdAt: "2022-01-04"
+    description: This One Operator is a Kubernetes custom controller whose goal is
```

not related to this PR but let reword the description a bit
```suggestion
    description: This operator is a Kubernetes custom controller whose goal is
```

---

Yeap, I already knew someone would say to divide them into two.! But, I wanted to make sure whether it is the matter of one pr with two commits or two different prs..! You mean two different commits in this pr, right?

And,,, it seems that release-0.23 branch includes the non-working yaml file.. Do we need to do something for that branch also?

---

it's good now. can you take my suggestion in 25eaf91 as well?

---

@mythi I did.! What do you think about deployments/operator/samples/deviceplugin_v1_xxxdeviceplugin.yaml files?

---

What about them?

---

@mythi If I understood correctly,, I had to change all samples/yaml files, makefile, and webhook_common.go also..!?
If not, please let me know.!

---

samples and webhook_common.go are not related to this work so they could go separate from this PR or at least in their own commit